python/examples/quickstarters/calendar_agent/crewai/main.py

```diff
@@ -1,30 +1,42 @@
 # Import base packages
 import os
 from datetime import datetime
+from typing import List, Tuple
 
 from composio_crewai import App, ComposioToolSet
 from crewai import Agent, Task, Crew
+from crewai.agent import Agent as CrewAgent
+from crewai.task import Task as CrewTask
 from dotenv import load_dotenv
 from langchain_openai import ChatOpenAI
 
 
-# Load environment variables
+# Load environment variables and verify they exist
 load_dotenv()
 
+if not os.getenv("OPENAI_API_KEY"):
```

Comment:
Good addition of environment variable validation. This will help users identify missing credentials early and provide clear error messages.